src/components/ViewDeployTargets/index.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import Modal from "components/Modal";
+import withLogic from "components/ViewDeployTargets/logic";
+import {StyledViewDeployTargets, StyledViewDeployTargetsModal} from "./StyledViewDeployTargets"
+
+
+/**
+ * Displays the value of the selected variable.
+ */
+export const ViewDeployTargets = ({
+  deployTargetNo,
+  deployTargets,
+  open,
+  openModal,
+  closeModal,
+}) => {
+  return (
+    <StyledViewDeployTargets>
+      <div className="margins">
+        <a className="hover-state" onClick={openModal}>
+          <label>{deployTargetNo}</label>
```

Comment:
rather than the number, maybe change it to `expand` or something? the number made it not super obvious it was a clickable link

Reply:
or as discussed, move deploytargets to a side nav tab if they are present, and make this link to the tab too